includes/models/model.llms.lesson.php

```diff
@@ -720,190 +719,216 @@ public function has_content() {
 		}
 	}
 
+
+
+
+
 	/**
 	 * Get next lesson ID
 	 *
 	 * @since 1.0.0
 	 * @since 3.24.0
 	 * @since 4.4.0 Improve query so that unpublished siblings do not break expected results.
-	 * @since [version] Use a numeric comparison for the next position meta query.
+	 * @since [version] Use a numeric comparison for the previous position meta query.
+	 * @since [version] Refactor to use helper method `get_sibling()`.
 	 *
-	 * @return false|int ID of the next lesson, if any, `false` otherwise.
+	 * @return false|int WP_Post ID of the next lesson or `false` if one doesn't exist.
 	 */
 	public function get_next_lesson() {
 
-		// phpcs:disable WordPress.DB.SlowDBQuery.slow_db_query_meta_key
-		// phpcs:disable WordPress.DB.SlowDBQuery.slow_db_query_meta_query
-		// phpcs:disable WordPress.WP.PostsPerPage.posts_per_page_posts_per_page
+		return $this->get_sibling( 'next' );
+
+	}
+
+	/**
+	 * Get previous lesson ID
+	 *
+	 * @since 1.0.0
+	 * @since 3.24.0 Unknown.
+	 * @since 3.24.0 Unknown.
+	 * @since 4.4.0 Improve query so that unpublished siblings do not break expected results.
+	 *              Use strict comparisions where needed.
+	 *              Make sure to always return `false` if no previous lesson is found.
+	 * @since [version] Use a numeric comparison for the previous position meta query.
+	 * @since [version] Refactor to use helper method `get_sibling()`.
+	 *
+	 * @return false|int WP_Post ID of the previous lesson or `false` if one doesn't exist.
+	 */
+	public function get_previous_lesson() {
+
+		return $this->get_sibling( 'prev' );
+
+	}
+
+	/**
+	 * Retrieve the sibling lesson in a specified direction.
+	 *
+	 * @since [version]
+	 *
+	 * @param string $direction Direction of navigation. Accepts either "prev" or "next".
+	 * @return false|int WP_Post ID of the sibling lesson or `false` if one doesn't exist.
+	 */
+	protected function get_sibling( $direction ) {
+
+		$lesson = $this->get_sibling_lesson_query( $direction );
```

Comment:
I'm not a fan of these "get_*_query" method names. The first time I read them I thought they would returned a WP_Query or even an SQL query.

What do you think if we turn `get_sibling_lesson_query` into `query_sibling_lesson` for example?

Also `get_sibling_section_query` makes me think we're going to retrieve a section not a lesson, what do you think about something like `query_sibling_lesson_from_adjacent_section`?

Filter names then should be changed accordingly.